README.md

```diff
@@ -13,21 +13,49 @@ If you discover potential security issues, see the reporting instructions on our
 
 ## Build Locally
 
-This site is built using Jekyll 4.3.2 with Ruby 3.3.0. Other versions may work, but YMMV.
+This site is built with [Zola](https://www.getzola.org/).
```

Comment:
I can't comment on the line I want, but there is a mention Git submodules at the top of the Readme which is a lie now.